tests/integration/connectToken.ts

```diff
@@ -0,0 +1,111 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import Device from '../../lib/twilio/device';
+import Call from '../../lib/twilio/call';
+import { generateAccessToken } from '../lib/token';
+import { expectEvent } from '../lib/util';
+
+describe('connectToken', function() {
+  this.timeout(10000);
+
+  function waitFor(n: number, reject?: boolean) {
+    return new Promise((res, rej) => setTimeout(reject ? rej : res, n));
+  }
+
+  function toObject(map: Map<string, string>) {
+    return Array.from(map.keys()).reduce((result: Record<string, string>, key: string) => {
+      result[key] = map.get(key)!;
+      return result;
+    }, {})
+  }
+
+  let callerDeviceA: Device;
+  let callerDeviceB: Device;
+  let receiverDevice: Device;
+  let reconnecDeviceA: Device;
```

Comment:
```suggestion
  let reconnectDeviceA: Device;
```
Reconnect is misspelled in several places in this file.